specs-go/v1/manifest_list.go

```diff
@@ -50,8 +50,10 @@ type ManifestDescriptor struct {
 	Platform Platform `json:"platform"`
 }
 
-// ManifestList  references manifests for various platforms.
+// ManifestList references manifests for various platforms.
+// This structure provides `application/vnd.oci.image.manifest.list.v1+json` mediatype when marshalled to JSON.
 type ManifestList struct {
+	specs.MediaTyped
 	specs.Versioned
```

Comment:
If `Versioned` contains `MediaTyped` (which is what you currently have), isn't it redundant to list `MediaTyped` here?